src/analytics/formTracker/formTracker.ts

```diff
@@ -119,6 +119,36 @@ export class FormTracker extends BaseTracker {
     }
     return label;
   }
+  /**
+   * Get the section value from the label or legend associated with the HTML form element.
+   *
+   * @param {FormField[]} elements - The array of form fields.
+   * @return {string} The label or legend of the field.
+   */
+  getSectionValue(elements: FormField[]): string {
```

Comment:
You don't need to give an array of elements. Just your field is enough.
Because now we are parsing our formfield array before (in the new for loop), we will always only have 1 element.

Reply:
ok yep , that make sense have pushed the changes now